outwatch/src/main/scala/outwatch/util/Store.scala

```diff
@@ -18,7 +18,7 @@ object Store {
    * @tparam A The Action Type
```

Comment:
This `@param reducer` comment needs to be removed.

Reply:
Github won't let me comment on the actual line. It is right above the one I highlighted.

Reply:
Thank you, fixed in 6929127